test/component/uk/gov/hmrc/individualsemploymentsapi/v2/EmploymentsSpec.scala

```diff
@@ -1,73 +0,0 @@
-/*
- * Copyright 2021 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package component.uk.gov.hmrc.individualsemploymentsapi.v2
-
-import component.uk.gov.hmrc.individualsemploymentsapi.stubs.BaseSpec
-import play.api.http.Status._
-import play.api.libs.json.Json.parse
-import scalaj.http.Http
-import uk.gov.hmrc.individualsemploymentsapi.sandbox.v1.SandboxData.sandboxMatchId
-
-class EmploymentsSpec extends BaseSpec {
-  private val payeEmploymentsScope = "read:individuals-employments-paye"
```

Comment:
Same here. This is an `Employments` spec not a sandbox spec. If we remove this spec entirely do we still have coverage of the remaining prod route?

Reply:
Spec only had 1 test in it pertaining to the sandbox Api in V2. Hence this was removed.

Reply:
My question then is do we need to keep the test and target the remaining controller? I.e. switch over from sandbox? 

If you feel we still have the coverage without this then fine. I simply wanted to pose the question.

Reply:
We have other component test for that controller (see test/component/uk/gov/hmrc/individualsemploymentsapi/controller/v2/LiveEmploymentsControllerSpec.scala)

Reply:
👍